dot_lpdev_cmds.sh

```diff
@@ -204,14 +204,14 @@ function __lpdev_geth_init {
       "eip158Block": 3,
       "byzantiumBlock": 4,
       "clique": {
-        "period": 2,
+        "period": 1,
```

Comment:
does this change mean that geth should approve blocks every second instead of every 2 seconds?

Reply:
Yep - I didn't see any reason to not have quicker block times in the dev environment, but let me know if you think there are.

Reply:
I like one-second blocks!